recipe-server/package.json

```diff
@@ -109,14 +111,14 @@
     "redux-devtools-log-monitor": "1.0.11",
     "redux-logger": "3.0.6",
     "redux-mock-store": "1.1.4",
-    "sass-loader": "4.0.1",
+    "sass-loader": "6.0.6",
     "style-loader": "0.13.1",
     "stylelint": "7.6.0",
     "stylelint-config-standard": "15.0.0",
     "stylelint-order": "0.2.2",
-    "webpack": "1.13.2",
+    "uglifyjs-webpack-plugin": "^0.4.6",
```

Comment:
If we get better compression with uglify we should definitely go with it. I just thought @Osmose ended up also seeing better compression with babili. But maybe that was because he was specifically compressing ES6 content.